app/facades/git_hub_repos_facade.rb

```diff
@@ -1,18 +1,26 @@
 class GitHubReposFacade
+  attr_reader :repos
+
   def initialize(login, search_term)
     @login = login
     @search_term = search_term
+    @repos = fetch_repos
   end
 
-  def repos
+  def fetch_repos
     service = GitHubService.new
     @repos_data ||= service.find_repos(login, search_term)
 
+    return [] unless @repos_data[:items]
```

Comment:
I like the refactor here to use an unless statement. That way we are always checking.